### DISCUSSION TITLE
please pay attion to my suggestings drayor or at least tell me its not planned

### DISCUSSION THREAD

**Discussion by Natedogwoooof** [Draylar/identity#610] [Ideas]
dude i dont like rushing modders but you have unapproved poll requests all u need to do is approve them and upload the mod to curse

**Comment by Draylar** [comment]
Hi Nate, thanks for all your feedback in suggestions over the past year, I do regularly check out your posts and ideas and appreciate you greatly - just please keep in mind I have a job which results in me having extremely limited time/bandwidth to work on my free-time projects at this point. I will get to it when I get to it, but I can't promise anything in the short term. Thanks for understanding!

**Reply by Natedogwoooof** [reply]
understandable  mad respect for makeing free mods
